CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## Developer Changelog
+**July 4, 2023 - [DESCW-1219](https://apps.itsm.gov.bc.ca/jira/browse/DESCW-1219)**
+- Frontend
```

Comment:
Just want to say I appreciate how this project has a little more specificity in its CHANGELOG(frontend/backend). If you're ever trying to track down a bug or previous work in the commit history, it's always helpful to have more hints as to where it could be